docs/intro/template.md

```diff
@@ -6,34 +6,15 @@ sidebar_label: Create App from scratch
 
 ### Start new project using Graphback and GraphQL CLI
 
-The best way to start with Graphback is to choose one of the project boilerplates 
-available as part of official GraphQL CLI. 
-GraphQL CLI is an very poular command line tool that integrates Graphback and 
-other community libraries offering a comprehensive way to work with GraphQL APIs.
-You can use any of the available GraphQL CLI templates to bootstrap Graphback enabled project. 
+The best way to start with Graphback is to choose one of the project boilerplates.
+You can use any of the available templates to bootstrap Graphback enabled project or create your own
 
 ### Procedure
 
-With yarn:
 ```
-yarn global add graphql-cli@canary
-```
-
-With npm:
-```
-npm install -g graphql-cli@canary
-```
-
-Execute init command to create project from scratch
-
-```
-graphql init
+npx graphback-cli init
```

Comment:
This doesn't work. It should be `npx graphback init <project-name>`

Reply:
It works but init is not released (yet)

Reply:
npx graphback-cli config

Reply:
Oh okay. So CLI will be its own thing (unlike before).

Still though, do you need to still provide the name immediately?

I get:

```
graphback init
graphback init <name>

Create Graphback project from available templates
```

Reply:
Yes.. That is confusion related to intallation process.

npx is package based. npm install -g graphback-cli and then graphback init is the same, but npx is much better to recomend as it will always make sure you have latest version. Globally installed one can also lead to many issues ( you can use your dev version etc.) Wheverever we want to teste released version it should be npx and we will have global one linked to our dev env